Best Practices/Challenges/UnitTests/UnitTests.Challenge/Calculator.cs

```diff
@@ -0,0 +1,40 @@
+﻿namespace UnitTests.Challenge
+{
+    /// <summary>
+    /// These parts involve writing tests for parts 1-4, follow the document for more information
+    /// </summary>
+    public class Calculator
+    {
+        private int _total;
+
+        public Calculator(int startValue)
+        {
+            _total = startValue;
```

Comment:
suggestion: Do we have a part to unit test the constructor setting the start value or just that the methods use the start value in their calculations?